admin/src/main/ts/src/app/management/structure-informations/structure-informations.component.ts

```diff
@@ -179,6 +186,44 @@ export class StructureInformationsComponent extends OdeComponent implements OnIn
 
   duplicateSettings(): void
   {
+    let uaiArray: string[] = this.settings.uaiList.match(/[0-9]{7}[a-zA-Z]/g);
+    let uaiSize = uaiArray.length;
+    if (uaiSize != 0) {
+      this.infoService.checkUAIs(this.structure.id, uaiArray).subscribe(
+        {
+          next: (data) =>
+          {
+            if (data.length != uaiSize) // means there are non-existent UAIs on the list
+            {
+              let uais = data.map(struc => struc.UAI);
+              this.settings.lightboxTitle = this.bundles.translate("management.structure.informations.duplication.setting.lightbox.error.title");
+              this.settings.lightboxList = uaiArray.filter(uai => !uais.includes(uai.toUpperCase()));
+              this.settings.lightboxMessage = this.bundles.translate("management.structure.informations.duplication.setting.lightbox.error.message");
+              this.settings.lightboxCanValidate = false;
+            }
+            else
+            {
+              this.settings.lightboxTitle = this.bundles.translate("management.structure.informations.duplication.setting.lightbox.title");
+              this.settings.lightboxMessage = this.bundles.translate("management.structure.informations.duplication.setting.lightbox.message")
+              .replace("{{name}}", this.structure.name);
```

Comment:
la méthode translate de sijil peut prendre en 2e argument un paramètre pour la clé i18n, exemple :
this.bundlesService.translate(`group.card.class.${group.filter}`, {name: group.classes[0].name});

plutôt que d'uiliser le .replace